### ISSUE TITLE
Further Documentation of Protocol & Help

### ISSUE THREAD

**Issue by illuzn** [bajarrr/rheem_eziset#1]
Damn, I wish I'd found this before my multiple days journey of trying to figure out how all this works.

Would have been a massive headstart. I've gone into some more depth documenting the protocol here: https://illuzn.github.io/Rheem-Eziset-Protocol/

Hope this helps.

Is this component still under active development? I note you don't have any licence attached to this if I wish to fork and continue on with your excellent work.

**Comment by bajarrr** [comment]
That's some really good, in depth documentation you've done there.

This is my first open source project, and I had overlooked things like a license, so thanks for that, I've now added one.

I got the project to a very basic state that just works and then life got busy and I haven't been able to develop further. I also hadn't realised it'd been 5 months. I'm veey happy for you to run with it and develop it properly.
